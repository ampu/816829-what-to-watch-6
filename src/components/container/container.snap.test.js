import React from 'react';

import {render} from '@testing-library/react';

import Container from './container';

describe(`Container component`, () => {
  it(`Should render according to the snapshot`, () => {
    const {container} = render(<Container/>);
    expect(container).toMatchSnapshot();
  });

  it(`Should render according to the snapshot while extra properties provided`, () => {
    const {container} = render(<Container isAbsolute isCentered>fakeChildren</Container>);
    expect(container).toMatchSnapshot();
  });
});
