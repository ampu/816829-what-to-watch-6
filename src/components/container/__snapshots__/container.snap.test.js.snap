// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component Should render according to the snapshot 1`] = `
<div>
  <div
    class="container"
  />
</div>
`;

exports[`Container component Should render according to the snapshot while extra properties provided 1`] = `
<div>
  <div
    class="container container--centered container--absolute"
  >
    fakeChildren
  </div>
</div>
`;
